dev/MRTCore/TestApps/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/MainWindow.h

```diff
@@ -10,7 +10,7 @@
 
 #pragma pop_macro("GetCurrentTime")
 
-#include <MRM.h>
+//#include <MRM.h>
```

Comment:
delete?

Reply:
This will be back.

Right now, this sample gets its MRT bits from WinUI. WinUI does not ship our flat C APIs and so this does not work. Once we move to WinUI ingesting MRTCore via the Reunion package this code will be pertinent. It does look kind of odd to have that much commented out code, but I don't want to lose it

Reply:
#ifdef ?

Reply:
That is a good idea! Also acts as an explanation of sorts. I will do that throughout.